mem/demo/3d-individual-demo.input

```diff
@@ -1,73 +0,0 @@
-dimension			3
-units				lj
-atom_style			hybrid sphere dipole
-boundary			p p p
-
-read_data			"3d-individual-demo.info"
-
-
-group 				mem     type 1
-group				vehicle type 2
-group				ligand 	type 3:22
-
-group				np      type 2:22
-
-
-velocity np set 0 0 -2
-
-#membrane parameters
-variable        rc           equal    2.6
-variable        rmin         equal    1.122462
-variable        mu           equal    3
-variable        zeta         equal    4
-variable        eps          equal    4.34
-variable        sigma        equal    1.00
-variable        theta0_11    equal    0
-
-pair_style		hybrid/overlay 		membrane ${rc}	lj/cut 5.04
-
-#membrane-np interactions	
-pair_coeff		*	*	lj/cut		0.0			0.0	0.0
-pair_coeff		1	2	lj/cut		100.0			4.0	4.45
-pair_coeff		1	3	lj/cut		10.0	1	2.0
-pair_coeff		1	4	lj/cut		10.0	1	2.0
-pair_coeff		1	5	lj/cut		10.0	1	2.0
-pair_coeff		1	6	lj/cut		10.0	1	2.0
-pair_coeff		1	7	lj/cut		10.0	1	2.0
-pair_coeff		1	8	lj/cut		10.0	1	2.0
-pair_coeff		1	9	lj/cut		10.0	1	2.0
-pair_coeff		1	10	lj/cut		10.0	1	2.0
-pair_coeff		1	11	lj/cut		10.0	1	2.0
-pair_coeff		1	12	lj/cut		10.0	1	2.0
-pair_coeff		1	13	lj/cut		10.0	1	2.0
-pair_coeff		1	14	lj/cut		10.0	1	2.0
-pair_coeff		1	15	lj/cut		10.0	1	2.0
-pair_coeff		1	16	lj/cut		10.0	1	2.0
-pair_coeff		1	17	lj/cut		10.0	1	2.0
-pair_coeff		1	18	lj/cut		10.0	1	2.0
-pair_coeff		1	19	lj/cut		10.0	1	2.0
-pair_coeff		1	20	lj/cut		10.0	1	2.0
-pair_coeff		1	21	lj/cut		10.0	1	2.0
-pair_coeff		1	22	lj/cut		10.0	1	2.0
-
-pair_modify		pair 	lj/cut		shift yes
-
-#membrane-membrane interactions
-pair_coeff		*	*	membrane     0.0           0.0            0.0           0.0       0.0      0.0    0.0
-pair_coeff		1	1	membrane     ${eps}        ${sigma}       ${rmin}       ${rc}     ${zeta}  ${mu}  ${theta0_11}
-
-neigh_modify	every 1 delay 1
-
-fix		fLANG	    all	    langevin 1.0 1.0 1.0 1000 zero yes
-fix		fNPH  	    mem     nph/sphere x 0.0 0.0 10.0 y 0.0 0.0 10.0 &
-		couple xy  update dipole
-fix_modify	fNPH  press thermo_press
-fix		fRIGID		np      rigid/nve group 1 np
-
-dump			id all xyz 100 3d-individual-demo.xyz
-
-thermo_style    custom step pe ke etotal	
-
-timestep       0.01
-thermo         100
-run            50000
```

Comment:
I think we should keep this file somewhere, for the reasons mentioned above. I appreciate its use cases are a little unclear, but I use it a LOT.

Reply:
False alarm, it still exists :)